Source/TailwindTraders.Mobile/TailwindTraders.Mobile/Features/Settings/DefaultSettings.cs

```diff
@@ -14,7 +14,19 @@ public static class DefaultSettings
 
         public const bool UseFakeAuthentication = DebugMode;
 
-        public const bool ForceAutomaticLogin = DebugMode;
+        public const bool ForceAutomaticLogin =
+#if IS_UI_TEST
```

Comment:
Could we introduce a new feature toggling for that? The goal is avoiding adding #ifs further than at one single place, like DebugMode does.